operatorconfig/driverconfig/powerflex/v2.6.0/controller.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: <DriverDefaultReleaseName>-controller
-  namespace: <DriverDefaultReleaseNamespace>
+  namespace: <DriverDefaultReleaseName>
```

Comment:
It is not DriverDefaultReleaseNamespace. It is DriverDefaultReleaseName only. Since the git is comparing 2.3. with 2.6 yamls it is showing as changed. But when you check 2.5 yamls it is DriverDefaultReleaseName  only.

Reply:
why are the samples' version wrong? 
I don't follow you @rensyct